src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -43,14 +44,27 @@ public Fixer()
 		public void Fix(DirectoryInfo directory)
 		{
 			WriteLine("Current directory: " + directory);
-			var xproj = GetXproj(directory);
+		    featureFiles = directory.GetFiles("*.feature", SearchOption.AllDirectories);
+		    var generatedFileExists = featureFiles.Where(f => !File.Exists(f.FullName + ".cs")).ToList();
+
+            var xproj = GetXproj(directory);
```

Comment:
I use the editorconfig extension so would certainly work for me. To be fair, for small projects like this I tend to use VSCode which automatically uses editorconfig files I believe (or perhaps I installed something and forgot...).